### ISSUE TITLE
/boot/config.txt has moved to /boot/firmware/config.txt

### ISSUE THREAD

**Issue by openoms** [raspiblitz/raspiblitz#4505]
on the v0.11.0RC5 image:
```
sudo cat /boot/config.txt 
DO NOT EDIT THIS FILE

The file you are looking for has moved to /boot/firmware/config.txt



dtoverlay=waveshare35a:rotate=90
```

```
sudo cat /boot/firmware/config.txt 
# For more options and information see
# http://rptl.io/configtxt
# Some settings may impact device functionality. See link above for details

# Uncomment some or all of these to enable the optional hardware interfaces
#dtparam=i2c_arm=on
#dtparam=i2s=on
#dtparam=spi=on

# Enable audio (loads snd_bcm2835)
# dtparam=audio=on

# Additional overlays and parameters are documented
# /boot/firmware/overlays/README

# Automatically load overlays for detected cameras
camera_auto_detect=1

# Automatically load overlays for detected DSI displays
display_auto_detect=1

# Automatically load initramfs files, if found
auto_initramfs=1

# Enable DRM VC4 V3D driver
dtoverlay=waveshare35a:rotate=90
max_framebuffers=2

# Don't have the firmware create an initial video= setting in cmdline.txt.
# Use the kernel's default instead.
disable_fw_kms_setup=1

# Run in 64-bit mode
arm_64bit=1

# Disable compensation for displays with overscan
disable_overscan=1

# Run as fast as firmware / board allows
arm_boost=1

[cm4]
# Enable host mode on the 2711 built-in XHCI USB controller.
# This line should be removed if the legacy DWC2 controller is required
# (e.g. for USB device mode) or if USB support is not required.
otg_mode=1

[all]


# Raspiblitz
kernel=kernel8.img
max_usb_current=1
dtparam=nvme
dtoverlay=waveshare35a:rotate=90
dtoverlay=waveshare35a:rotate=90
```

will need to check the build_sdcard entries

**Comment by openoms** [comment]
@rootzoll I only have the
```
dtoverlay=waveshare35a:rotate=90
```

added to the 
```
/boot/firmware/config.txt
```
on rc6.

Need to double check this.

```
kernel=kernel8.img
```
is needed for Fulcrum

**Comment by openoms** [comment]
still getting 
```
getconf PAGE_SIZE
16384
```
attempt to fix in #4529

**Comment by rootzoll** [comment]
all other places config.txt has moved are addressed in the PR #4530

**Comment by rootzoll** [comment]
OK this looks good now - closing for final release
